lib/components/user/styled.ts

```diff
@@ -46,8 +46,25 @@ export const TripHeader = styled.h3`
   margin-top: 0px;
 `
 
+export const TripPannelTitle = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`
+
 export const TripPanelHeading = styled(Panel.Heading)`
   background-color: white !important;
+  border-color: #fff !important;
+  padding: 20px 25px 0 25px;
+
+  button {
+    background: transparent;
+    border: 0;
+  }
```

Comment:
Is this needed, or can this replace a styled-button defined elsewhere?

Reply:
This was changed to a link, so this styling has been removed!